packages/accordion/demo/stories/data.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+export const SECTIONS = Array(5)
+  .fill(undefined)
+  .map((_, i) => i);
```

Comment:
Can we switch this up to be something more like https://github.com/zendeskgarden/react-containers/blob/8a30d4b802ae3042b5f1abfffac39683f2dedb35/packages/selection/demo/stories/data.ts#L8? I think `['section-1', 'section-2' ...]` will read better in the story control.